docs/guide/custom-ui.md

```diff
@@ -245,7 +245,7 @@ Do not use core actions as your `action` property value - this would lead to unp
 * You may mark the action as "dangerous" via an optional property `modifiers: [ 'danger' ]` (see the 'Archive' and 'Unpublish' menu items).
 * An additional optional boolean property `manuallyPublished` is supported. When set to true, the custom menu item is available only for document types that do not have the `autopublish: true` or `localized: false` options set.
 * The `conditions` property is optional. It takes an array of one or more strings specifying conditions that all must be satisfied to determine if the action can be run on the current doc. Valid values are: 'canPublish', 'canEdit', 'canDismissSubmission', 'canDiscardDraft', 'canLocalize', 'canArchive', 'canUnpublish', 'canCopy', 'canRestore'
-* The optional `moduleName` property can be used to invoke the action API on a module other than the defining module.
+* The optional `moduleName` property can be used to pass a module name other than `self.name` to the component.
```

Comment:
Sorry Bob, this is still not accurate, I know I keep belaboring this point... how about: "The optional `moduleName` property can be used to override the `moduleName` prop passed to the modal. By default it will be the name of the piece type module corresponding to the individual piece, or `@apostrophecms/page` in the case of pages."

Reply:
Corrected to your text.